test/functional/feature_coloredcoin.py

```diff
@@ -432,6 +434,22 @@ def run_test(self):
 
         test_transaction_acceptance(node, TxFailure8, accepted=False, reason=b'invalid-colorid')
 
+        #TxSuccess9 - (UTXO-17) - issue 10 REISSUABLE1, change
+        txSuccess9 = CTransaction()
+        txSuccess9.vin.append(CTxIn(COutPoint(txSuccess7.malfixsha256, 1), b""))
+        txSuccess9.vout.append(CTxOut(10, script_reissuable1))
+        txSuccess9.vout.append(CTxOut(4999999649, change_script))
+        sig_hash, err = SignatureHash(txSuccess7.vout[1].scriptPubKey, txSuccess9, 1, SIGHASH_ALL)
+        signature = self.coinbase_key.sign(sig_hash) + b'\x01'
+        txSuccess9.vin[0].scriptSig = CScript([signature])
```

Comment:
@Naviabheeman Why are you setting your signature here?
It looks like the code after this is changing the output value and re-signing it.

Reply:
the signature is needed to calculate the size of the transaction. We need to calculate the fee so that the amount can be set to 'min transaction fee - 2'. Only then the error is actually generated every time the test is run. When I make the amount an absolute integer the test fails intermittently  if the signature size is different.

Reply:
I see.